style.css

```diff
@@ -15,3 +15,15 @@
     font-family: Inter, system-ui, sans-serif
   }
 }
+
+.divider {
```

Comment:
We should avoid class style in tailwind world. Try to extract them as component, for example

```tsx
const Divider = function () {
    return (
        <div className="relative w-full">
            <hr className="..." />
        </div>
    )
}
```

Reply:
[Create <Divider/> instead of class](https://github.com/pilagod/waallet-extension/pull/152/commits/8e4d5969cefb06494c74bdd5af81978ad3ceb7f0)